lib/functions/since.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class Since
+  def self.calculate_units
+    ->(from, to, unit) { get_diff_of_date(from, to, unit) }
```

Comment:
you do not convert `from` and `to` to time

Reply:
Thanks. I implemented conversion.